### PR TITLE
HIPAA Landing Page: Fix contact button, redirect from /hipaa-compliance.

### PR DESCRIPTION
**Pull request by josh-padnick**
Previously clicking the "I'm interested" button went to a 404. This PR fixes that.

In addition, there is now a redirect from `/hipaa-compliance` to `/hipaa-compliance-on-aws`.

### CODE REVIEW

File: pages/landing/hipaa/pricing/_price-col.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h2>Special early access pricing</h2>
       </div>
       <div data-mh="plans-pricing-bottom">
         <p>
-          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">I'm interested</a>
+          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance-on-aws/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">I'm interested</a>
```
Comment (ebeneliason) [inline pages/landing/hipaa/pricing/_price-col.html:17]:
```suggestion
          <a id="pricing-cta-aws" class="btn btn-info pricing-ctas" href="/hipaa-compliance-on-aws/contact" ga-on="click" ga-event-category="{{ page.path | slugify }}" ga-event-action="select-cta">Join the waitlist</a>
```

In the thread we talked about changing this to a stronger CTA, like the suggestion, or even just "Request Early Access". It looks like that hasn't actually happened yet. Do you want to sneak it in here?

**Review by ebeneliason — APPROVED** [review]
Looks good to me. I wish there were a better solution for relative paths in links to avoid breaking things if the permslinks change, but this looks correct as it is now.

File: pages/landing/hipaa/pricing/_price-col.html (hunk 1 above)
Comment (josh-padnick) [reply pages/landing/hipaa/pricing/_price-col.html:17]: Sure, "request early access" sounds great, so I'll go with that.

Comment (josh-padnick) [reply pages/landing/hipaa/pricing/_price-col.html:17]: Actually, we've already got that as a link on the homepage, so I'll go with your suggestion.

**Pull request merged**